### DISCUSSION TITLE
Mode of a dataset

### DISCUSSION THREAD

**Discussion by romanstetsyk** [stdlib-js/stdlib#1014] [stdlib-bot Q&A] [answered]
Is there a package that computes a dataset's mode (the most frequent element)?

[a comment by an automated account is not shown]

**Reply by romanstetsyk** [reply]
/ask Is there a package that computes a mode of an array of strings/numbers?

[a comment by an automated account is not shown]

**Comment by Planeshifter** [comment]
Hey @romanstetsyk, while we don't have a capacity in `stdlib` yet to calculate the mode of a dataset, you may try the following package of ours if it's sufficient for you to calculate the mode of an array:

https://github.com/compute-io/mode

**Comment by romanstetsyk** [comment]
@Planeshifter That's exactly what I need; thanks a lot!
I also noticed that there are a lot of packages in `compute` org with similar functionality to `stdlib`.
